Automattic/sensei
# Add task Create your First Course

Fixes #5636

### Changes proposed in this Pull Request

* Add class to handle the task "Create your First Course" in Sensei Home;
* Also add tests;

### Testing instructions

On a WP installation containing a version of Sensei based on this branch without any courses:

1. Check that "Create your first course" task is returned in `/wp-json/sensei-internal/v1/home` in your WP installation;
2. Confirm that task has "done" as `false` by default;
3. Import the demo course;
4. Verify that the task still has "done" set to `false`;
5. Now, create another course with any name;
6. Verify that the task now has "done" set to `true`;
